contracts/script/EigenDADeployer.s.sol

```diff
@@ -226,6 +242,42 @@ contract EigenDADeployer is DeployOpenEigenLayer {
             )
         );
 
+        eigenDAThresholdRegistryImplementation = new EigenDAThresholdRegistry();
+
+        VersionedBlobParams[] memory versionedBlobParams = new VersionedBlobParams[](0);
```

Comment:
Does it make sense to hardcode the params for version 0 here if we're going to use the same values across all environments?